notebooks/1-welcome.ipynb

```diff
@@ -13,24 +13,102 @@
+    "- **some experience with Unix commands**\n",
+    "  - Jupyter Book is a command line tool, commend for which will by used in this tutorial in Jupyter Notebooks.\n",
+    "  - Though not required for this tutorial, you can check this beginner's friendly [Unix training material by The Carpentries](https://swcarpentry.github.io/shell-novice/)\n",
+    "- **an *optional module* uses an example file with Python scripts**\n",
+    "  - Learners who use other programming language can replace Python script with short script or code snippets in programming language of their preference"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "👍 **Prerequisite:**\n",
-    "- prior knowledge of using GitHub for project repositories. \n",
-    "- Basic experience working with Markdown files and Jupyter notebooks will be a plus. \n",
-    "- One **optional module** shows example using Python scripts\n",
+    "👋 Who we are\n",
+    "---\n",
+    "\n",
+    "**Instructors**\n",
+    "\n",
+    "- **Martina G. Vilas**: Core maintainer of _The Turing Way_, Open Source contributor of Jupyter Book, PhD student at Max Plank Institute, Germany. *Twitter: [@martinagvilas](https://twitter.com/martinagvilas)*\n",
+    "- **Sarah Gibson**: Core maintainer of _The Turing Way_ and Binder, contributor of Project Jupyter, Research Software Engineer, The Alan Turing Institute, UK. *Twitter: [@drsarahlgibson](https://twitter.com/drsarahlgibson)*\n",
+    "- **Malvika Sharan**: Community Manager - _The Turing Way_, Research Associate, Whitaker Lab, The Alan Turing Institute, UK. *Twitter: [@malvikasharan](https://twitter.com/malviaksharan)*\n",
+    "\n",
+    "**Project leads**\n",
+    "\n",
+    "- **Kirstie Whitaker**: Project lead - _The Turing Way_, head of the Tools, Practices, and Systems research programme, The Alan Turing Institute, UK. *Twitter: [@kirstie_j](https://twitter.com/kirstie_j)*\n",
+    "- **Chris Holdgraf**: Project lead - Jupyter Book, member of Project Jupyter and Binder, fellow at the Berkeley Institute for Data Science, USA. *Twitter: [@choldgraf](https://twitter.com/choldgraf)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "👍 Tutorial Setup\n",
+    "---\n",
+    "\n",
+    "*This setup guide will take you from reading this page online to reading it locally.*\n",
+    "\n",
+    "**Jupyter Notebooks**\n",
+    "\n",
+    "This tutorial will make use of Jupyter Notebooks to guide the lessons (divided into short modules). \n",
+    "Please [install Anaconda](https://www.anaconda.com/), a Python distribution that comes with everything we need for the lesson. \n",
+    "Detailed installation instructions for various operating systems can be found in [Anaconda documentation](https://docs.anaconda.com/anaconda/install).\n",
```

Comment:
Should we recommend [miniconda](https://docs.conda.io/en/latest/miniconda.html) instead? It's more lightweight than full anaconda.

Reply:
That's a good point. I can add that too.